2024-04-01 17:50:44,554 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Current SDK version is 0.16.5
2024-04-01 17:50:44,554 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Configure stats pid to 4705
2024-04-01 17:50:44,554 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Loading settings from /Users/viditpokharna/.config/wandb/settings
2024-04-01 17:50:44,554 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Loading settings from /Users/viditpokharna/Downloads/assignment-4/wandb/settings
2024-04-01 17:50:44,554 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-01 17:50:44,554 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-01 17:50:44,554 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-01 17:50:44,554 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-01 17:50:44,554 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-01 17:50:44,554 INFO    MainThread:4705 [wandb_init.py:_log_setup():527] Logging user logs to /Users/viditpokharna/Downloads/assignment-4/wandb/run-20240401_175044-dv9qnc79/logs/debug.log
2024-04-01 17:50:44,554 INFO    MainThread:4705 [wandb_init.py:_log_setup():528] Logging internal logs to /Users/viditpokharna/Downloads/assignment-4/wandb/run-20240401_175044-dv9qnc79/logs/debug-internal.log
2024-04-01 17:50:44,555 INFO    MainThread:4705 [wandb_init.py:init():567] calling init triggers
2024-04-01 17:50:44,555 INFO    MainThread:4705 [wandb_init.py:init():574] wandb.init called with sweep_config: {}
config: {}
2024-04-01 17:50:44,555 INFO    MainThread:4705 [wandb_init.py:init():592] re-initializing run, found existing run on stack: iop4dmcv
2024-04-01 17:50:44,556 INFO    MainThread:4705 [wandb_run.py:_finish():2052] finishing run viditdpokharna/scene-recognition/iop4dmcv
2024-04-01 17:50:44,556 INFO    MainThread:4705 [jupyter.py:save_history():445] not saving jupyter history
2024-04-01 17:50:44,557 INFO    MainThread:4705 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-01 17:50:44,557 INFO    MainThread:4705 [wandb_init.py:_jupyter_teardown():455] cleaning up jupyter logic
2024-04-01 17:50:44,557 INFO    MainThread:4705 [wandb_run.py:_atexit_cleanup():2292] got exitcode: 0
2024-04-01 17:50:44,558 INFO    MainThread:4705 [wandb_run.py:_restore():2275] restore
2024-04-01 17:50:44,558 INFO    MainThread:4705 [wandb_run.py:_restore():2281] restore done
2024-04-01 17:51:22,505 INFO    MainThread:4705 [wandb_run.py:_footer_history_summary_info():3920] rendering history
2024-04-01 17:51:22,507 INFO    MainThread:4705 [wandb_run.py:_footer_history_summary_info():3952] rendering summary
2024-04-01 17:51:22,516 INFO    MainThread:4705 [wandb_run.py:_footer_sync_info():3879] logging synced files
2024-04-01 17:51:22,622 INFO    MainThread:4705 [wandb_init.py:init():617] starting backend
2024-04-01 17:51:22,622 INFO    MainThread:4705 [wandb_init.py:init():621] setting up manager
2024-04-01 17:51:22,627 INFO    MainThread:4705 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-04-01 17:51:22,627 INFO    MainThread:4705 [wandb_init.py:init():629] backend started and connected
2024-04-01 17:51:22,643 INFO    MainThread:4705 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-04-01 17:51:22,643 INFO    MainThread:4705 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-04-01 17:51:22,643 INFO    MainThread:4705 [wandb_init.py:init():721] updated telemetry
2024-04-01 17:51:22,645 INFO    MainThread:4705 [wandb_init.py:init():754] communicating run to backend with 90.0 second timeout
2024-04-01 17:51:23,787 INFO    MainThread:4705 [wandb_run.py:_on_init():2344] communicating current version
2024-04-01 17:51:23,941 INFO    MainThread:4705 [wandb_run.py:_on_init():2353] got version response 
2024-04-01 17:51:23,941 INFO    MainThread:4705 [wandb_init.py:init():805] starting run threads in backend
2024-04-01 17:51:26,455 INFO    MainThread:4705 [wandb_run.py:_console_start():2323] atexit reg
2024-04-01 17:51:26,456 INFO    MainThread:4705 [wandb_run.py:_redirect():2178] redirect: wrap_raw
2024-04-01 17:51:26,456 INFO    MainThread:4705 [wandb_run.py:_redirect():2243] Wrapping output streams.
2024-04-01 17:51:26,456 INFO    MainThread:4705 [wandb_run.py:_redirect():2268] Redirects installed.
2024-04-01 17:51:26,456 INFO    MainThread:4705 [wandb_init.py:init():848] run started, returning control to user process
2024-04-01 17:51:26,457 INFO    MainThread:4705 [wandb_run.py:_config_callback():1347] config_cb None None {'learning_rate': 0.001, 'epochs': 5, 'batch_size': 32}
2024-04-01 18:07:44,289 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Current SDK version is 0.16.5
2024-04-01 18:07:44,289 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Configure stats pid to 4705
2024-04-01 18:07:44,289 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Loading settings from /Users/viditpokharna/.config/wandb/settings
2024-04-01 18:07:44,289 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Loading settings from /Users/viditpokharna/Downloads/assignment-4/wandb/settings
2024-04-01 18:07:44,289 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-01 18:07:44,289 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-01 18:07:44,289 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-01 18:07:44,289 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-01 18:07:44,289 INFO    MainThread:4705 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-01 18:07:44,289 INFO    MainThread:4705 [wandb_init.py:_log_setup():527] Logging user logs to /Users/viditpokharna/Downloads/assignment-4/wandb/run-20240401_180744-5f9pxdfy/logs/debug.log
2024-04-01 18:07:44,290 INFO    MainThread:4705 [wandb_init.py:_log_setup():528] Logging internal logs to /Users/viditpokharna/Downloads/assignment-4/wandb/run-20240401_180744-5f9pxdfy/logs/debug-internal.log
2024-04-01 18:07:44,290 INFO    MainThread:4705 [wandb_init.py:_jupyter_setup():473] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x28769c070>
2024-04-01 18:07:44,290 INFO    MainThread:4705 [wandb_init.py:init():567] calling init triggers
2024-04-01 18:07:44,290 INFO    MainThread:4705 [wandb_init.py:init():574] wandb.init called with sweep_config: {}
config: {}
2024-04-01 18:07:44,290 INFO    MainThread:4705 [wandb_init.py:init():592] re-initializing run, found existing run on stack: dv9qnc79
2024-04-01 18:07:44,291 INFO    MainThread:4705 [wandb_run.py:_finish():2052] finishing run viditdpokharna/scene-recognition/dv9qnc79
2024-04-01 18:07:44,291 INFO    MainThread:4705 [wandb_run.py:_atexit_cleanup():2292] got exitcode: 0
2024-04-01 18:07:44,292 INFO    MainThread:4705 [wandb_run.py:_restore():2275] restore
2024-04-01 18:07:44,292 INFO    MainThread:4705 [wandb_run.py:_restore():2281] restore done
2024-04-01 18:08:09,164 INFO    MainThread:4705 [wandb_run.py:_footer_history_summary_info():3920] rendering history
2024-04-01 18:08:09,167 INFO    MainThread:4705 [wandb_run.py:_footer_history_summary_info():3952] rendering summary
2024-04-01 18:08:09,172 INFO    MainThread:4705 [wandb_run.py:_footer_sync_info():3879] logging synced files
